packages/web-forms/e2e/test-cases/markdown.test.ts

```diff
@@ -0,0 +1,76 @@
+import { expect, Locator, Page, test } from '@playwright/test';
+import { FillFormPage } from '../page-objects/pages/FillFormPage.ts';
+import { PreviewPage } from '../page-objects/pages/PreviewPage.ts';
+
+const ANIMATION_TIME = 50;
+
+async function selectMultiDropdownOption(page: Page, container: Locator, ...labels: string[]) {
+	const dropdown = container.locator('.multi-select-dropdown');
+	await dropdown.click();
+	for (const label of labels) {
+		await page.locator(`.p-multiselect-option:has-text("${label}")`).click();
+	}
+	await page.waitForTimeout(ANIMATION_TIME);
+}
+
+async function selectDropdownOption(page: Page, container: Locator, label: string) {
+	const dropdown = container.locator('.dropdown');
+	await dropdown.click();
+	await page.locator(`.p-select-option:has-text("${label}")`).click();
+	await page.waitForTimeout(ANIMATION_TIME);
+}
```

Comment:
These functions can be part of a [SelectControl](https://github.com/getodk/web-forms/tree/main/packages/web-forms/e2e/page-objects/controls), we'll probably need them in the future

Reply:
Done.